tropopause_find/tropopause_find.meta

```diff
@@ -0,0 +1,299 @@
+[ pi ]
+  standard_name = pi_constant
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ errmsg ]
+  standard_name = ccpp_error_message
+  units = none
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+[ errflg ]
+  standard_name = ccpp_error_code
+  units = 1
+  type = integer
+  dimensions = ()
+  intent = out
+
+[ccpp-arg-table]
+  name  = tropopause_find_run
+  type  = scheme
+[ ncol ]
+  standard_name = horizontal_loop_extent
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ pver ]
+  standard_name = vertical_layer_dimension
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ lat ]
+  standard_name = latitude
+  units = radians
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ pint ]
+  standard_name = air_pressure_at_interface
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_interface_dimension)
+  intent = in
+[ pmid ]
+  standard_name = air_pressure
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ t ]
+  standard_name = air_temperature
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ zi ]
+  standard_name = geopotential_height_wrt_surface_at_interface
+  units = m
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_interface_dimension)
+  intent = in
+[ zm ]
+  standard_name = geopotential_height_wrt_surface
+  units = m
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ phis ]
+  standard_name = surface_geopotential
+  units = m2 s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ calday ]
+  standard_name = fractional_calendar_days_on_end_of_current_timestep
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = ()
+  intent = in
+[ tropp_p_loc ]
+  standard_name = tropopause_air_pressure_from_climatology
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, number_of_months_in_year)
+  intent = in
+[ tropp_days ]
+  standard_name = tropopause_calendar_days_from_climatology
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = (number_of_months_in_year)
+  intent = in
+[ tropLev ]
+  standard_name = model_level_number_at_tropopause
+  units = 1
+  type = integer
+  dimensions = (horizontal_loop_extent)
+  intent = out
+[ tropP ]
+  standard_name = tropopause_air_pressure
+  units = Pa
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = out
+[ tropT ]
+  standard_name = tropopause_air_temperature
+  units = K
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = out
+[ tropZ ]
+  standard_name = tropopause_altitude
```

Comment:
Just noting that we will need to change the standard names in this file as well, as discussed in previous review comments and in the standard names spreadsheet.

Reply:
Thanks for the reminder, updated throughout based on CCPP standard names spreadsheet.